docs/supported_inventory_types.md

```diff
@@ -153,6 +153,7 @@ See the docs on [how to add a new Extractor](/docs/new_extractor.md).
 | Stripe Webhook Secret                       | `secrets/stripewebhooksecret`        |
 | Tink keyset                                 | `secrets/tinkkeyset`                 |
 | Vapid keys                                  | `secrets/vapidkey`                  |
```

Comment:
Nit: Would you mind aligning the | at the end for Vapid and reCAPTCHA with the above ones?